### PR TITLE
docs: dev update 63

### PR DESCRIPTION
**Pull request by delta1**
adds development update 63

### CODE REVIEW

File: _updates/2021-11-11-update-63.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+[Covenants](https://bitcoinops.org/en/topics/covenants/) are a method of placing restrictions on how a given [transaction output](https://rfc.tari.com/Glossary.html#unspent-transaction-outputs) can be spent. For example: you may want to specify that the original NFT creator receives a commission on each subsequent sale, or that Bob's crypto vault can only be spent to a specific address belonging to his niece.
+
+Work is progressing on [a draft for covenants implementation](https://gist.github.com/sdbondi/035ca1db7980d974a99e914e6f499a97) on Tari, which will soon be published as an RFC for community feedback.
+
+Barring any complications, implementation will start after the RFC period!
+
+### BTC Atomic Swaps
+
+We recently published an [RFC for implementing BTC - XTR atomic swaps](https://rfc.tari.com/RFC-0240_AtomicSwap.html#description). These are "trustless" transactions that enable users to swap Tari (XTR) for Bitcoin (BTC) or vice versa, without requiring a 3rd party.
```
Comment (stringhandler) [inline _updates/2021-11-11-update-63.md]:
```suggestion
We recently published an [RFC for implementing BTC - XTR atomic swaps](https://rfc.tari.com/RFC-0240_AtomicSwap.html#description). These are trustless transactions that enable users to swap Tari (XTR) for Bitcoin (BTC) or vice versa, without requiring a 3rd party.
```

File: _updates/2021-11-11-update-63.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- [`wallet_ffi`](https://github.com/tari-project/tari/tree/development/base_layer/wallet_ffi)
+- [`wallet-android`](https://github.com/tari-project/wallet-android)
+- [`wallet-ios`](https://github.com/tari-project/wallet-ios)
+
+### Hack on the future of Digital Assets @ Tari
+
+As always, come join us to chat about ongoing development in the `dev` channel on our [Discord server](https://discord.gg/q3Sfzb8S2V), or on IRC in the `#tari-dev` channel on the [Libera.Chat](https://libera.chat) IRC network.
+
+Tari Labs is hiring! View the open positions at [jobs.lever.co/tari](https://jobs.lever.co/tari).
```
Comment (stringhandler) [inline _updates/2021-11-11-update-63.md]:
```suggestion

```

**Review by stringhandler — COMMENTED** [review]
Looks good. Just a few edits

**Comment by delta1** [comment]
Thanks @mikethetike updated, I tried to keep you as `co-authored-by` but then the commit was only "partially verified"

File: _updates/2021-11-11-update-63.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+layout: update
+tag: Developer Update
+date: 2021-11-11
+author: Byron Hambly
+thumbnail: update-background.jpg
+title: Tari Development Update
+class: subpage
+---
+
+## Are we there yet?
```
Comment (CjS77) [inline _updates/2021-11-11-update-63.md:11]:
```suggestion
## Are we there yet?

Things are really starting to buzz in the Tari development space. 

We have the usual array of bug fixes and performance improvements. We also report on progress that's been made on multiple fronts, including Covenants, cross-chain atomic swaps and the provisionally-named DAN validator nodes.
```

File: _updates/2021-11-11-update-63.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+As described in [Wen Mainnet?]({% link _updates/2021-09-28-update-61.md %}), Tari has a complete set of Layer 1 features.
+
+The current development focus is full steam ahead on Layer 2 features for the Digital Assets Network ([DAN](https://rfc.tari.com/RFC-0300_DAN.html#related-requests-for-comment)).
+
+## What are we working on?
```
Comment (CjS77) [inline _updates/2021-11-11-update-63.md]:
```suggestion
## What are we working on?

As described in [Wen Mainnet?]({% link _updates/2021-09-28-update-61.md %}), Tari has a complete set of Layer 1 features.

The current development focus is full steam ahead on Layer 2 features for the Digital Assets Network ([DAN](https://rfc.tari.com/RFC-0300_DAN.html#related-requests-for-comment)).


```

**Review by CjS77 — APPROVED** [review]

**Comment by delta1** [comment]
Thanks @CjS77

**Pull request merged**